assignments/objects.js

```diff
@@ -17,29 +17,81 @@ const example = {
   "gender": "F"
 }
 
+
+
 // Write your intern objects here:
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 2,
+  "name": "kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function() {
+    console.log("Hello, my name is Kennan");
```

Comment:
```suggestion
    return "Hello, my name is Kennan";
```
You want to return the statement instead of using console.log so when you console.log it on challenge 3, the console you use in challenge 3 has a value to return, right now it will try to console.log something with no value